simple_model:
    joint_state_controller:
        type: joint_state_controller/JointStateController
        publish_rate: 20

    joint_1_position_controller:
        type: effort_controllers/JointPositionController
        joint: joint_1
        pid: {p: 1.0, i: 1.0, d: 0.0}

    joint_2_velocity_controller:
        type: effort_controllers/JointVelocityController
        joint: joint_2
        pid: {p: 1.0, i: 1.0, d: 0.0}

    joint_3_velocity_controller:
        type: effort_controllers/JointVelocityController
        joint: joint_3
        pid: {p: 100.0, i: 1.0, d: 10.0}

    joint_4_velocity_controller:
        type: effort_controllers/JointVelocityController
        joint: joint_4
        pid: {p: 100.0, i: 1.0, d: 10.0}